From 0fc3c803c34311eb05c5c7a7e710c8591b592649 Mon Sep 17 00:00:00 2001
From: Thierry Bordaz <tbordaz@redhat.com>
Date: Thu, 27 Jul 2017 18:10:05 +0200
Subject: [PATCH] Ticket 49313 - Change the retrochangelog default cache size

Bug Description:
    Default retroCL backend entry cache size is 2Mb.
    It has been reported in many deployments that DB corruption could
    be prevented by increasing entry cache to 200Mb.
    There is no identified reproducible steps to debug this DB corruption.
    So to prevent this problem we are increasing the entry cache

Fix Description:
    Set default cn=changelog cache to 200Mb (based on production cases)
    An other option would be to set a maximum number of entries but
    as we do not know if it works to prevent DB corruption, let's prefere
    entry cache size

https://pagure.io/389-ds-base/issue/49313

Reviewed by: William Brown

Platforms tested: F23

Flag Day: no

Doc impact: no
---
 ldap/servers/plugins/retrocl/retrocl.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/retrocl/retrocl.h b/ldap/servers/plugins/retrocl/retrocl.h
index 6963d4b..eef1a17 100644
--- a/ldap/servers/plugins/retrocl/retrocl.h
+++ b/ldap/servers/plugins/retrocl/retrocl.h
@@ -58,7 +58,7 @@ typedef struct _cnumRet {
 #else
 #define RETROCL_DLL_DEFAULT_THREAD_STACKSIZE 131072L
 #endif
-#define RETROCL_BE_CACHEMEMSIZE  "2097152"
+#define RETROCL_BE_CACHEMEMSIZE  "209715200"
 #define RETROCL_BE_CACHESIZE "-1"
 #define RETROCL_PLUGIN_NAME "DSRetroclPlugin"
 
-- 
2.9.4

